src/rev_ai/models/revaiapi_deployment_config_constants.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+class RevAiApiDeployment(Enum):
+    US = 'US'
+    EU = 'EU'
+
+RevAiApiDeploymentConfigMap = {

```

Comment:
just curious, why do we need this map? why just not follow usual pattern with options classes?

```
RevAiStreamingClient(..., RevAiStreamingClientOptions(....))
```

we can even define a few static preconfigured deployments: RevAiStreamingClientOptions.US, RevAiStreamingClientOptions.EU and pass them or something like this.

Reply:
i want to reuse this for any type of revai service, not just streaming